lib/src/cose.dart

```diff
@@ -239,7 +180,7 @@ class Cose {
             npk,
             Uint8List.view(signers.buffer, 0, signers.length),
             sigStructureBytes.buffer.asUint8List());
-        print(verified);
+        CoseLogger.print(verified);
       } else if (-35 == a) {

```

Comment:
Give these values a name -35 -> sha384, -36 -> sha512
Also a better variable name than `a` would be nice

Reply:
Yes indeed. I would like to update that as well. For now this pull requests splits up the library in multiple files & added an extra method `extractKid` from a pem string. But this should be updated in the future. As well as updating the analysis_options.yaml